README.md

```diff
@@ -1,67 +1,17 @@
 # Cluster API
-## What is the Cluster API?
 
-The Cluster API is a Kubernetes project to bring declarative, Kubernetes-style
-APIs to cluster creation, configuration, and management. It provides optional,
-additive functionality on top of core Kubernetes.
+Cluster API provides the ability to manage Kubernetes supportable hosts in the
+context of OpenShift.
 
-Note that Cluster API effort is still in the prototype stage while we get
-feedback on the API types themselves. All of the code here is to experiment with
-the API and demo its abilities, in order to drive more technical feedback to the
-API design. Because of this, all of the prototype code is rapidly changing.
+This branch contains an implementation of the OpenShift machinset-controller
+and OpenShift machine-controller, and shared code for various OpenShift
+Cluster API Actuators.
 
-![Cluster API Architecture](architecture.png "Cluster API Architecture")
+Each component is deployed and controlled by the OpenShift [machine-api-operator](https://github.com/openshift/machine-api-operator)
 
-To learn more, see the [Cluster API KEP][cluster-api-kep].
+# Upstream Implementation
+Other branches of this project may more closely track the upstream
```

Comment:
Other branches of this repository may choose to track the upstream Kubernetes Cluster API project.